e2e-test-app/pom.xml

```diff
@@ -52,24 +73,62 @@
       <groupId>com.sap.ai.sdk</groupId>
       <artifactId>orchestration</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
+      <artifactId>cloudplatform-connectivity</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
+      <artifactId>cloudplatform-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.sap.cloud.sdk.datamodel</groupId>
+      <artifactId>openapi-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-autoconfigure</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-context</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-web</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty.toolchain</groupId>
+      <artifactId>jetty-jakarta-servlet-api</artifactId>
+    </dependency>
```

Comment:
(Question)

Are you sure this is the correct dependency? (For what API?)

Reply:
`jetty-jakarta-servlet-api` is the replacement for `spring-boot-starter-web`. It's needed to run the `Application extends SpringBootServletInitializer`

Reply:
yep looks odd, why would spring need a replacement and why would it be some servlet dependency? I only know the `extends SpringBootServletInitializer` from WAR deployment, which gives me strong feelings 😉

Reply:
Should we update it in the Cloud SDK archetype as well?

Reply:
Huh, I wasn't aware it's in there. Maybe it is still needed for some edge case in there? If you want you could try it, but I guess it's not a big deal if we leave it in